packages/integrations/vercel/src/index.ts

```diff
@@ -130,6 +130,20 @@ function getAdapter({
 			i18nDomains: 'experimental',
 			envGetSecret: 'stable',
 		},
+		client: {
+			internalFetchHeaders: skewProtection
+				? (): Record<string, string> => {
+						const deploymentId = process.env.VERCEL_DEPLOYMENT_ID;
+						if (deploymentId) {
+							return { 'x-vercel-deployment-id': deploymentId };
```

Comment:
```suggestion
							return { 'x-deployment-id': deploymentId };
```

See the docs here https://vercel.com/docs/skew-protection